wiki/Community/Projects/puush/en.md

```diff
@@ -0,0 +1,42 @@
+---
+tags:
+  - puu.sh
+  - puush.me
+  - puushme
+---
+
+# puush
+
+::: Infobox
+![Screenshot of puush's interface](img/puushScreenshot.png "Screenshot of puush's application interface, as taken from CNET's coverage of the service")
+:::
+
+**[puush.me](https://puush.me)**, often referred to as **puush**, is a file sharing service run and hosted by ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2) and ::{ flag=AU }:: [nekodex](https://osu.ppy.sh/users/102). First launched in June 2010, the service serves as a successor to [upppy](/wiki/upppy) with a greater focus on screen sharing and user data management.
+
+## Service
+
+puush allows for users to upload files and screenshots to their personal accounts using a dedicated [Windows](https://puush.me/dl/puush-installer.exe), [Mac OS](https://puush.me/dl/puush.zip), or [iOS](https://itunes.apple.com/au/app/puush/id386524126?mt=8) application. Once uploaded, these files can then be managed from the puush.me website, put into "pools" (photo albums), or shared elsewhere using an easily shareable `puu.sh` link.
+
+A core feature of puush is its ability to capture and share screenshots instantly, which makes it particularly popular among [modders](/wiki/Modding). As puush is intended for sharing files, not as a permanent file storage, uploaded files are only retained for a limited amount of time before they are deleted.
+
+## Subscription
+
+While puush was available to everyone free of charge, in order to keep the service self-sufficient, puush offered a "Pro" plan that users could subscribe to for US$15/year.[^puush-pro-plan-ref] This plan came with a couple of extra benefits for the subscribing user, namely:
+
+- Increased maximum filesize limit (20 MB for Free users vs. 250 MB for Pro users)
+- Increased file retention period (1 months since last access for Free users vs. 6 months since last access for Pro users)
+
+Similar to osu!, all puush users are not subjected to any ads regardless of their subscription status.
+
+## Downscaling
+
+Starting from March 2014, puush no longer accepts registrations for the Pro plan as PayPal suspended puush's account due to a suspected Terms of Service breach.[^puush-paypal-suspension-ref] This prompted ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2) to concurrently pay out around $400 per month out of his own pocket to keep the service running, as puush's financials are independent from osu! and not tied to [supporter tag](/wiki/osu!supporter) purchases.[^puush-finances-ref]
```

Comment:
the part about financials being separate is weird since it's linking to a statement which doesn't really state that 🤷